src/dreamer/orchestrator.py

```
@@ -0,0 +1,205 @@
+                )
+                logger.info(
+                    f"[{run_id}] Targeting observations with surprisal range: "
+                    + f"{high_surprisal_obs[-1].surprisal:.3f} to {high_surprisal_obs[0].surprisal:.3f}"
+                )
+            else:
+                logger.info(
+                    f"[{run_id}] No high-surprisal observations found using standard probing questions"
+                )
+
+        except Exception as e:
+            logger.error(f"[{run_id}] Surprisal sampling failed: {e}", exc_info=True)
```

Ruff (0.14.10)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)